src/alerting/components/SearchBar.test.tsx

```diff
@@ -0,0 +1,62 @@
+// Libraries
+import React from 'react'
+import {screen, fireEvent, prettyDOM, cleanup} from '@testing-library/react'
+
+// Components
+import SearchBar from 'src/alerting/components/SearchBar'
```

Comment:
It's not bad thing to have redundant tests because it increases chance to catch errors.

Reply:
I do think there are some downsides, actually. Having duplicate tests means that there are higher maintenance costs to keep these tests passing. It also can unnecessarily increase the amount of time it takes for our test suite to run. I'm fine with leaving the `show/hide autocomplete` test in this PR but I think going forward we should focus on unit tests for the reducers instead of the UI components.